src/components/accordion/accordion.astro

```diff
@@ -7,7 +7,7 @@ export interface Props {
 const { title, id } = Astro.props;
 ---
 
-<details class="group mb-4">
+<details class="group mb-4" id={id} data-faq>
```

Comment:
> data-faq

Is this component for our faq page only? I can imagine it being used at different places